app/views/content/train-to-be-a-teacher/get-school-experience.md

```diff
@@ -5,6 +5,7 @@ subcategory: Experience life in the classroom
 description: |-
   Get school experience to explore what life is like in a primary or secondary classroom and find out if teaching is right for you.
 related_content:
+    Experience teaching with a paid internship : "/teaching-internship-providers"
```

Comment:
In the [content principles](https://educationgovuk.sharepoint.com/:p:/r/sites/TeacherServices/Shared%20Documents/General/Get%20Into%20Teaching%20Website%20Project/BETA/Content/Navigation/Archive/Content%20design%20standards%20and%20principles%20v1.pptx?d=w7977283cdbbb5be1bdc856c35057c08f&csf=1&web=1&e=lukOtr) you/Valentine put together a while ago, I'm pretty sure we talked about avoiding duplicate links on the same page - specifically, that the links in the related content section shouldn't repeat anything that's already on the page? which I do think makes sense still